linux_os/guide/services/ntp/service_timesyncd_configured/oval/shared.xml

```diff
@@ -0,0 +1,102 @@
+<def-group>
+  <definition class="compliance" id="{{{ rule_id }}}" version="1">
+    {{{ oval_metadata("Ensure that timesyncd is enabled and configured") }}}
+    <criteria comment="Timesyncd is enabled and configured" operator="AND">
+      <criterion test_ref="{{{ rule_id }}}_test_systemd_timesyncd_service_enabled" />
+      <criteria comment="Timesyncd is enabled and configured" operator="AND">
+        <criteria comment="Timesyncd is enabled and configured" operator="OR">
+          <criterion test_ref="{{{ rule_id }}}_test_systemd_timesyncd_master_ntp_configuration" />
+          <criterion test_ref="{{{ rule_id }}}_test_systemd_timesyncd_dropin_ntp_configuration" />
+        </criteria>
+        <criteria comment="Timesyncd is enabled and configured" operator="OR">
```

Comment:
I believe the `RootDistanceMax` option deserves its own rule. Smaller and simpler rules for a single purpose makes the rules more granular and reusable. Please, consider splitting this different parameter.

Reply:
:+1: added in [60563f8](https://github.com/ComplianceAsCode/content/pull/10670/commits/60563f8314278d4fc62734781efc76fe2b0691c6)